### PR TITLE
feat: 🎸 Add handler to map MultiSig events

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Adds handler to map multisig events

### Breaking Changes

NA

### JIRA Link

DA-903

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: db/genesisMigrations.ts [hunk 1]
```diff
@@ -99,7 +103,49 @@ export const genesisMigrationQueries = async (): Promise<string[]> => {
     .map(getInserts)
     .flat();
 
+  allAccounts = allAccounts.concat([...systematicIssuerIdentities, ...gcIdentities]);
+
+  const multiSigEntries = await api.query.multiSig.multiSigToIdentity.entries();
```
[the remark this answers is not in the document]
Comment (prashantasdeveloper) [reply db/genesisMigrations.ts]: Nice catch. Even though it wouldn't have affected as inserting is skipped if entry is already present but its more performant to update only genesis hash data. I have updated accordingly

**Review by polymath-eric — COMMENTED** [review]
lgtm, but IDK why its getting stuck on block 200 in the tests

**Review by polymath-eric — APPROVED** [review]
lgtm

File: schema.graphql [hunk 2]
```diff
@@ -2194,3 +2194,102 @@ type ChildIdentity @entity {
...
+  signaturesRequired: Int!
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+"""
+Represents MultiSig signer status
+"""
+enum MultiSigSignerStatusEnum {
+  Authorized ## when a signer is added
```
Comment (polymath-eric) [inline schema.graphql:2225]: (nit) I think `Pending` is a bit more clear. `Authorized` makes it seem like the account is already added to me. Not a big deal either way.

Comment (prashantasdeveloper) [reply schema.graphql:2225]: I wanted to keep it as similar as the chain

**Review by polymath-eric — APPROVED** [review]

**Pull request merged**